hieradata/class/backend.yaml

```diff
@@ -38,6 +38,7 @@ govuk::node::s_base::apps:
   - hmrc_manuals_api
   - imminence
   - kibana
+  - local-links-manager
```

Comment:
This needs to be `local_links_manager` to match https://github.com/alphagov/govuk-puppet/blob/add-local-links-manager-app/modules/govuk/manifests/apps/local_links_manager.pp#L12